config/rbac/leader_election_role.yaml

```diff
@@ -6,22 +6,29 @@ metadata:
 rules:
 - apiGroups:
   - ""
-  - coordination.k8s.io
   resources:
   - configmaps
-  - leases
   verbs:
+  - create
```

Comment:
This entire file is auto-generated by operator-sdk when operator was created. Not sure why it specified these permissions in the first place. Wondering if we should keep it as is, unless these wouldn't be needed for sure.

Reply:
Removing those permissions doesn't seem to cause any errors or warnings in the log but there could be areas we are not hitting in our tests. I'll remove the changes in this file for now.